.github/CODEOWNERS

```diff
@@ -130,7 +130,7 @@
 #/avm/res/signal-r-service/web-pub-sub/ @Azure/avm-res-signalrservice-webpubsub-module-owners-bicep @Azure/avm-core-team-technical-bicep
 #/avm/res/sql/managed-instance/ @Azure/avm-res-sql-managedinstance-module-owners-bicep @Azure/avm-core-team-technical-bicep
 /avm/res/sql/server/ @Azure/avm-res-sql-server-module-owners-bicep @Azure/avm-core-team-technical-bicep
-#/avm/res/storage/storage-account/ @Azure/avm-res-storage-storageaccount-module-owners-bicep @Azure/avm-core-team-technical-bicep
+/avm/res/storage/storage-account/ @Azure/avm-res-storage-storageaccount-module-owners-bicep @Azure/avm-core-team-technical-bicep
```

Comment:
Please sdd team to `avm-technical-reviewers-bicep`

Reply:
I think @fblix already created the group, but @matebarabas needs to approve it or something

Reply:
The team is created but is not a member of avm-technical-reviewers-bicep. Thus it does not have write permissions, thus the codeowners file contains errors.

Reply:
This may be - but what is the expectation now? It's not my team. Can I add a team that I do not own to another team? As said earlier, I anyways expect that @fblix already did that and Mate needs to approve. 

That said, I think it's fair to say that this has to happen before the PR can be merged - so we can keep this comment as blocking until then.

Reply:
A  note on this. The UI can bug so even if it shows it does not have a parent (and does not show pending) then the team isn't configured. That being said @AlexanderSehr I am not sure of the permission on @Azure/avm-technical-reviewers-bicep but you should be able to add teams to this if you have the right permission on the group

Reply:
@eriqua & @ChrisSidebotham seems valid now (i.e., ready for review)